config/application.rb

```diff
@@ -23,6 +24,9 @@ class Application < Rails::Application
     )
     IdentityConfig.build_store(configuration)
 
+    AssetSources.manifest_path = Rails.root.join('public', 'packs', 'manifest.json')
+    AssetSources.cache_manifest = Rails.env.production?
```

Comment:
@mitchellhenke Thoughts on caching in `Rails.env.test?` as well? Might help a little with feature tests in CI.

Reply:
```suggestion
    AssetSources.cache_manifest = Rails.env.production? || Rails.env.test?
```

Reply:
I think might as well. We can always roll it back.